packages/twenty-front/src/modules/ui/field/display/components/ActorDisplay.tsx

```diff
@@ -28,17 +32,23 @@ export const ActorDisplay = ({
       case 'IMPORT':
         return IconCsv;
       case 'EMAIL':
-        return IconGmail;
+        return context?.provider === ConnectedAccountProvider.MICROSOFT
```

Comment:
this way, there is no need of a feature flag. it falls back to current version if the provider is not provided